app/controllers/users_controller.rb

```diff
@@ -1,7 +1,7 @@
 class UsersController < ApplicationController
   def index
     @user = current_user
-    @users = User.all
+    @users = User.order(:id)
   end
 
   def show
```

Comment:
- [ ] Here, if a user is not logged in, this happens when trying to read posts:

![image](https://user-images.githubusercontent.com/57421823/167761516-0e1c77b3-8231-458e-ab73-f3c5efe4965e.png)

Please, correct this behavior.

Reply:
Addressed!